### ISSUE TITLE
npm ERR! code ERR_INVALID_PROTOCOL npm ERR! Protocol "https:" not supported. Expected "http:" 

### ISSUE THREAD

**Issue by buildingidgr** [stackblitz/bolt.new#2676]
### Describe the bug

This error appears all the time. 

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-cmkyue

### Steps to reproduce

1. Go to my project
2. Try to run it

### Expected behavior

Should run the code of my project

### Platform

```
Browser name = Safari
Full version = 18.1
Major version = 18
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.1 Safari/605.1.15
performance.memory = undefined
Username = buildingidgr
Chat ID = df1ce6e86ba4
```

**Comment by endocytosis** [comment]
Hi @buildingidgr, 

Are you able to set your project to "Private (accessible by URL)" by clicking the lock icon next to the project title so we can take a closer look? Thanks!

**Comment by buildingidgr** [comment]
DoneOn Nov 18, 2024, at 19:09, Brian ***@***.***> wrote:﻿
Hi @buildingidgr,
Are you able to set your project to "Private (accessible by URL)" by clicking the lock icon next to the project title so we can take a closer look? Thanks!

**Comment by kc0tlh** [comment]
@buildingidgr Bolt.new does not currently support Safari. We are adding a banner to indicate this soon, but in the meantime, please switch to Google Chrome or another Chromium based browser.

**Comment by buildingidgr** [comment]
Thank you!!!! I accidentally deleted a chat I made and spent like 10m tokens. Is it possible to undelete it? On Nov 20, 2024, at 01:50, Alexander Berger ***@***.***> wrote:﻿
Closed #2676 as completed.
